src/pkg/projects/netcoreappRIDs.props

```diff
@@ -33,5 +33,8 @@
     <UnofficialBuildRID Include="tizen.4.0.0-armel">
```

Comment:
Are both of these RIDs going to be supported going forward? Or another way of asking: Are you going to produce `runtime` packages for both `tizen.4.0.0` and `tizen.5.0.0` for every build of master?

Reply:
@eerhardt, yes, we'll use both RIDs